lib/index.ts

```diff
@@ -47,16 +50,26 @@ function searchBabylonFiles(root: string, currentPath: string, options: SearchOp
         scene.useDelayedTextureLoading = true;
         const doNotDelayLoadingForGeometries: string[] = [];
         const excludedMeshes = options.excludedMeshes;
+        const minMeshSize = options.minMeshSize;
         // Parsing meshes
         scene.meshes.forEach((mesh: any) => {
+
             if (!excludedMeshes.some(meshCheck => meshCheck.test(mesh.name))) {
-                // Do not delay load collisions object
-                if (mesh.checkCollisions) {
-                    if (mesh.geometryId) {
-                        doNotDelayLoadingForGeometries.push(mesh.geometryId);
+                const meshString = createDelayLoadingString(mesh, true);
+                if  (!minMeshSize || minMeshSize < meshString.length) {
+                    // Do not delay load collisions object
+                    if (mesh.checkCollisions) {
+                        if (mesh.geometryId) {
+                            doNotDelayLoadingForGeometries.push(mesh.geometryId);
+                        }
+                    } else {
+                        extract(mesh, currentPath, filename, true, meshString);
                     }
                 } else {
-                    extract(mesh, currentPath, filename, true);
+                    // tslint:disable-next-line:no-console
+                    console.log(
+                        `Skipping ${mesh.name} as size ${meshString.length} is smaller then ${minMeshSize}`,
```

Comment:
Typo, `then` instead of `than`. It would be nice to also mention where the number ${minMeshSize} comes from, e.g. `Skipping ${mesh.name} as size ${meshString.length} is smaller than ${minMeshSize} from minMeshSize option`